scripts/build

```diff
@@ -3,9 +3,12 @@ set -e
 
 source $(dirname $0)/version
 source $(dirname $0)/images
-
 cd $(dirname $0)/..
-
+curl -L -v https://github.com/replicatedhq/kots/releases/download/v1.101.0/kots_linux_amd64.tar.gz  --output kots_linux_amd64.tar.gz
```

Comment:
this is a workaround for not being able to navigate to the github user content from the docker compile step. This pulls the file in the compiling computer and sticks it where it is supposed to be, rather than downloading it inside of the image